config/layout-text.json

```diff
@@ -0,0 +1,21 @@
+{
+  "title": "Devfile.io",
+  "bannerTitle": "Devfiles",
+  "bannerBody": "An open standard defining containerized development environments that enables developer tools to simplify and accelerate workflows.",
+  "headerLinks": [
+    { "name": "Getting Started", "link": "/getting-started" },
+    { "name": "Docs", "link": "https://devfile.io/docs" },
```

Comment:
I dont think we should do this.

Currently, when the site is deployed in https://github.com/devfile/devfile.github.io, the docs is available in `/docs` (See build process for the mentioned repo https://github.com/devfile/landing-page/blob/main/.github/workflows/BuildAndDeploy.yaml#L30-L35)

We handled this previously via env `process.env.docsPath` in https://github.com/devfile/landing-page/blob/main/next.config.js#L20 and handled links accordingly. It would still work but i think we should use the relative path instead of the absolute path, since at the end of the day its in the same repo https://github.com/devfile/devfile.github.io

Reply:
For now is this fine? I would rather not add a bunch of logic when the next pr is going to completely change the docs/landing-page stuff.